incubator/bitcoincashd/templates/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.configurationFiles }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "bitcoincashd.fullname" . }}
```

Comment:
Add standard labels.

Reply:
Thank you! I have done all recommendations.